lib/appium/AppiumRunner.js

```diff
@@ -419,8 +411,9 @@ AppiumRunner.prototype.prepareApp = function () {
     var deviceString = self.options.device ? ' --device' : '';
     var buildCommand = 'cordova build ' + self.options.platform + deviceString;
 
+    // TODO: is removing medic.json still needed?
```

Comment:
Yes, this is still needed because we use this file to pass local medic server URI to paramedic plugin so it could report test results back to paramedic. We remove it before the Appium run so the test framework plugin won't start plugin tests right away.
So, basically this is needed to run Appium tests without running other plugin tests simultaneously.
This probably should've been mentioned by me in this comment 👼